app/assets/stylesheets/frontpage.css.scss

```diff
@@ -107,5 +132,9 @@ input#front-search-button {
 }
 
 .tag {
-  color: blue;
+  background: blue;
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.